### DISCUSSION TITLE
"The notebook took too long to render"

### DISCUSSION THREAD

**Discussion by bcoelho-leonardo** [community/community#32901] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,
I uploaded a Jupyter Notebook file to my repository, and this message is raised when trying to open it:

"The notebook took too long to render"

![image](https://user-images.githubusercontent.com/72246216/189902667-f3bcfc5e-a70d-41d9-888e-1dbca3fc6251.png)

The .ipynb file is only 14.1 MB ...

Has anyone else dealt with this issue?

Thanks

**Comment by jfullstackdev** [comment]
I was about to try it but it seems it's not public, do you have similar file that you can share ? so I can check the loading time too

**Reply by bcoelho-leonardo** [reply]
Dear @jdevstatic 
Thanks for replying
I cannot share the file due to confidential agreement

However, what I could add now is the following;
I can at least access the file content in the permalink, which makes this less of an issue!

**Comment by 3kh0** [comment, marked as answer]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Comment by eddietaylor** [comment]
I'm having the same issue and none of the proposed solutions seem to work. This is a big joke, considering the majority of the data science, OR, machine learning and science community use python via jupyter notebooks for development. 
![image](https://github.com/community/community/assets/68166349/22e3588a-f25b-4369-b97f-e05746137258)
